composer.json

```
@@ -8,13 +8,24 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^8.1",
         "guzzlehttp/guzzle": "^6.3",
         "ext-json": "*"
     },
     "autoload": {
         "psr-4": {
             "Demv\\GenericRequest\\": "src/"
         }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Tests\\Demv\\GenericRequest\\": "tests/"
```

Ist das nicht besser? Habe gesehen das die Frameworks diese Strategie haben :thinking: 
```suggestion
            "Demv\\GenericRequest\\Tests\\": "tests/"
```